### ISSUE TITLE
Graphql support

### ISSUE THREAD

**Issue by jlaughton87** [chasegiunta/craft-jason#118]
Hi, 

Im attempting to update your jason field with my JSON data (stringified) through graphql, but Im receiving the (unhelpful) error "Error: GraphQL error: Internal server error" when i call the mutation. 

Do you know of any reason why the JASON field will not accept a string type when populated using graphql?

Additional info:

If i change the fieldtype from Jason to a Plain Text field the mutation runs successfully (as expected) and the field is populated:

<img width="1061" alt="Screenshot 2021-02-04 at 17 16 10" src="https://user-images.githubusercontent.com/14894660/106929835-c22e1300-670c-11eb-8443-aaf613368e7e.png">

and then when i change the fieldtype back to Jason the data is displayed correctly:

![image](https://user-images.githubusercontent.com/14894660/106929947-e8ec4980-670c-11eb-8237-829db76bdcae.png)

So we know the data ima attempting to send is valid for your fieldtype.

**Comment by chasegiunta** [comment]
@jlaughton87 You're correct that Graphql support hasn't been added. I'm not familiar enough with Graphql to add it myself, but PRs are welcome.

**Comment by jlaughton87** [comment]
No worries, thanks for letting me know

**Comment by approached** [comment]
@jlaughton87 maybe this will help you https://github.com/chasegiunta/craft-jason/pull/136
